dev/tasks/python-wheels/github.osx.amd64.yml

```diff
@@ -68,10 +63,23 @@ jobs:
         shell: bash
         run: echo ${VCPKG_ROOT} >> $GITHUB_PATH
 
-      # workaround for https://github.com/apache/arrow/pull/11569#issuecomment-979592053
-      - name: Install OpenSSL
-        run: vcpkg install openssl
-
+      - name: Setup NuGet Credentials
+        shell: bash
+        env: 
+          GITHUB_TOKEN: {{ '${{ secrets.GITHUB_TOKEN }}' }}
+        run: |          
+          mono `vcpkg fetch nuget | tail -n 1` \
+          sources add \
+          -source "https://nuget.pkg.github.com/ursacomputing/index.json" \
```

Comment:
Can we use `${{ github.repository_owner }}` instead of `ursacomputing`?

Reply:
I didn't do this as to not pollute peoples repos with nuget packages but I guess this is an on demand job so it shouldn't be an issue :)

Reply:
I used the envvar version, is cleaner with jinja

Reply:
Ah of course sorry